main.go

```diff
@@ -35,6 +35,7 @@ import (
 
 func main() {
 	githubToken := flag.String("github-token", os.Getenv("GITHUB_TOKEN"), "Github access token")
+	assumeYes := flag.Bool("yes", false, "Automatic yes to prompts and run non-interactively.")
```

Comment:
Please rename this flag from yes to y in order to match the  "Press y to continue or n to abort"

Reply:
Done...